js/components/controlBar.js

```diff
@@ -22,7 +22,13 @@ var ControlBar = React.createClass({
     this.isMobile = this.props.controller.state.isMobile;
     this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
     this.moreOptionsItems = null;
-    return {};
+    this.vr = this.props.controller
+      && this.props.controller.getVrParams
```

Comment:
Can we also add a check to see if getVrParams is a function, since that is our expectation?